llvm/lib/CodeGen/MachineOutliner.cpp

```diff
@@ -121,6 +121,12 @@ static cl::opt<unsigned> OutlinerBenefitThreshold(
     cl::desc(
         "The minimum size in bytes before an outlining candidate is accepted"));
 
+static cl::opt<bool> OutlinerLeafDescendants(
+    "outliner-leaf-descendants", cl::init(true), cl::Hidden,
```

Comment:
I don't have a strong preference, but wonder if we want a disable flag instead, like `-disable-outliner-leaf-descendants` and we can drop `cl::init(false)`, or completely delete a flag while updating all tests. I'm all open for either option.